generators/chipyard/src/main/scala/TileOnlyConfig.scala

```diff
@@ -0,0 +1,207 @@
+package chipyard
+
+import chisel3._
+import chisel3.experimental.{IO, DataMirror}
+import chisel3.util._
+import org.chipsalliance.cde.config.{Parameters, Config, Field}
+import freechips.rocketchip.diplomacy._
+import freechips.rocketchip.interrupts._
+import chipyard.iobinders._
+import freechips.rocketchip.tile._
+import freechips.rocketchip.subsystem.RocketCrossingParams
+import freechips.rocketchip.tilelink._
+import testchipip._
+
+case object RocketTileOnly extends Field[RocketTileParams]
+
+
+
+class TileOnlyTestHarness(implicit p: Parameters) extends Module {
+  val io = IO(new Bundle {
+    val success = Output(Bool())
+  })
+  println("Elaborating TileOnlyTestHarness")
+  val lazyDut = LazyModule(new TileOnlyChipTop).suggestName("tileOnlyChipTop")
+  val dut = Module(lazyDut.module)
+}
+
+class TileOnlyChipTop(implicit p: Parameters) extends LazyModule
+  with BindingScope 
+  with HasIOBinders
+{
+  println("Elaborating TileOnlyChipTop")
+  lazy val lazySystem = LazyModule(p(BuildSystem)(p)).suggestName("TileOnlySystem")
+  lazy val module: LazyModuleImpLike = new LazyRawModuleImp(this) { }
```

Comment:
I think if you make lazySystem not lazy (just val lazySystem), and remove HasIOBinders, you won't need the println below.